nemoguardrails/eval/evaluate_moderation.py

```diff
@@ -91,7 +91,10 @@ def get_jailbreak_results(self, prompt, results):
             Task.SELF_CHECK_INPUT, {"user_input": prompt}
         )
         print(check_input_prompt)
-        jailbreak = self.llm(check_input_prompt)
+        try:
+            jailbreak = self.llm(check_input_prompt)
+        except ValueError:
+            jailbreak = ""
```

Comment:
We should inspect the error as well here. If we recognize it's an error that means a "block", then we should count that as a positive response for the jailbreak. Otherwise, re-raise.